2019/d01p1FuelCounterUpper.d

```diff
@@ -0,0 +1,7 @@
+unittest {
+  import std.algorithm;
+
+  auto input = [12, 14, 1969, 100756];
+  auto fuelTotal = input.map!"a / 3 - 2".sum;
+  assert(fuelTotal == 2 + 2 + 654 + 33583);
```

Comment:
And here we prove that.

Reply:
How can anyone else do so?

Reply:
Assuming my custom `d` DMD Docker image is built, `docker run --rm -v$PWD:/src d sh -c 'rdmd -unittest -main d01p1FuelCounterUpper.d`.

It does nothing.

Reply:
That's worth putting in the commit message.